### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by CaroD59**
Menu to access to all the pages

### CODE REVIEW

**Comment by LoicBrassart** [comment]
Need the SM's validation before I add mine

**Review by ThomasSupervil — DISMISSED** [review]
Everything seems good !

File: src/components/menu/Rules.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import CSSRules from './CSSRules';
+
+export default function Rules() {
+  return (
+    <CSSRules>
+      <div className="BigContainerRules">
+        <h1>Comment jouer ? // PARTIE EN PREPARATION //</h1>
```
Comment (LoicBrassart) [inline src/components/menu/Rules.jsx]: No need to say "Partie en préparation" here

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by ThomasSupervil — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
No more Evil

**Review by ThomasSupervil — DISMISSED** [review]

File: src/App.jsx [hunk 2]
```diff
@@ -1,15 +1,22 @@
+
```
Comment (LoicBrassart) [inline src/App.jsx]: Useless empty line

File: src/App.jsx [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
+import Cards from './components/Cards/Cards';
 import Play from './components/Play/Play';
+import MenuList from './components/menu/MenuList';
+
 
 export default function App() {
   return (
     <div className="App">
-      <BrowserRouter>
+    <BrowserRouter>
```
Comment (LoicBrassart) [inline src/App.jsx]: Indentation error

File: src/App.jsx [hunk 4]
```diff
@@ -1,15 +1,22 @@
...
         <MenuList />
          <Play />
       </BrowserRouter>
+    <Play />
```
Comment (LoicBrassart) [inline src/App.jsx]: Indentation error

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor errors to fix

**Pull request closed without merging**